src/libraries/Microsoft.Bcl.Memory/ref/Microsoft.Bcl.Memory.csproj

```
@@ -12,6 +12,18 @@
     <Compile Include="Microsoft.Bcl.Memory.Forwards.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' Or '$(TargetFramework)' == 'netstandard2.1'">
+    <Compile Include="Microsoft.Bcl.Memory.Forwards.ns21.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' Or '$(TargetFramework)' == 'netstandard2.0'">
```

nit/suggestion
```suggestion
  <ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))'">
```

It might be easier to read to express these as compatibility relationships.

---

Applied the suggestion. Thanks!